.gitignore

```diff
@@ -12,6 +12,7 @@
 /build
 
 # misc
+.vscode
 .DS_Store
```

Comment:
.env is missing here.

Reply:
I added the file, but VSCode does not gray it out and still pushes it up with it. Do I still need to manually delete the file on GitHub?

Reply:
Problem solved :)